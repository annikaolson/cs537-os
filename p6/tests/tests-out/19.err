Traceback (most recent call last):
  File "<string>", line 10, in <module>
OSError: [Errno 5] Input/output error
