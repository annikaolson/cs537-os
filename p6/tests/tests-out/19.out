Correct
allocated datablocks on /tmp/myranda/test-disk1: found 17 expected 18
